lib/src/screens/crypto_detail_screen.dart

```diff
@@ -1,10 +1,55 @@
 import 'package:flutter/material.dart';
 
+import '../constants/constants.dart' as constants;
+
 class CryptoDetailScreen extends StatelessWidget {
   const CryptoDetailScreen({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return const Text('CryptoDetailScreen');
+    double toolbarHeight = 70.0;
+    double backButtonHeight = toolbarHeight * .4;
+    double backButtonAndSymbolImageSpacing = 50.0;
+    double symbolImageWidth = 40.0;
```

Comment:
It looks like we might need to use these value again in the future in other screens. We should move them to constants

Reply:
They are used only for AppBar, so I will put them into CustomAppBar widget.